frontend/app/components/query/query.service.js

```diff
@@ -202,6 +202,37 @@ function QueryConstructorService(Filter, Sortation, UrlParamsHelper, INITIALLY_S
       this.columns = columns;
     },
 
+    applyDefaultsFromFilters: function(workPackage) {
```

Comment:
I disagree. The method has deep knowledge on API experimental attributes, not V3. We do have to remove it once the filters are rewritten, so its responsibility lies with the query.